packages/shared-types/src/Colors.ts

```diff
@@ -22,121 +22,106 @@
  * SOFTWARE.
  */
 
-type BaseColors = {
-  brand: string
-  link: string
-  electric: string
-  shamrock: string
-  barney: string
-  crimson: string
-  fire: string
-  licorice: string
-  oxford: string
-  ash: string
-  slate: string
-  tiara: string
-  porcelain: string
+type Primitives = {
   white: string
-}
+  grey11: string
+  grey12: string
+  grey14: string
+  grey24: string
+  grey45: string
+  grey57: string
+  grey70: string
+  grey82: string
+  grey100: string
+  grey125: string
+
+  blue12: string
+  blue45: string
+  blue57: string
+  blue70: string
+  blue82: string
+
+  green12: string
+  green45: string
+  green57: string
+  green70: string
+  green82: string
+
+  orange12: string
+  orange30: string
+  orange45: string
+  orange57: string
+  orange70: string
+  orange82: string
 
-type TextColors = {
-  textDarkest: BaseColors['licorice']
-  textDark: BaseColors['ash']
-  textLight: BaseColors['porcelain']
-  textLightest: BaseColors['white']
-  textBrand: BaseColors['brand']
-  textLink: BaseColors['link']
-  textAlert: BaseColors['barney']
-  textInfo: BaseColors['brand']
-  textSuccess: BaseColors['shamrock']
-  textDanger: BaseColors['crimson']
-  textWarning: BaseColors['fire']
+  red12: string
+  red45: string
+  red57: string
+  red70: string
+  red82: string
 }
 
-type BackgroundColors = {
-  backgroundDarkest: BaseColors['licorice']
-  backgroundDark: BaseColors['ash']
-  backgroundMedium: BaseColors['tiara']
-  backgroundLight: BaseColors['porcelain']
-  backgroundLightest: BaseColors['white']
-  backgroundBrand: BaseColors['brand']
-  backgroundBrandSecondary: BaseColors['oxford']
-  backgroundAlert: BaseColors['barney']
-  backgroundInfo: BaseColors['brand']
-  backgroundSuccess: BaseColors['shamrock']
-  backgroundDanger: BaseColors['crimson']
-  backgroundWarning: BaseColors['fire']
+type Contrasts = {
+  white: Primitives[keyof Primitives]
+  grey11: Primitives[keyof Primitives]
+  grey1214: Primitives[keyof Primitives]
+  grey1424: Primitives[keyof Primitives]
+  grey24: Primitives[keyof Primitives]
+  grey4570: Primitives[keyof Primitives]
+  grey5782: Primitives[keyof Primitives]
+  grey100: Primitives[keyof Primitives]
+  grey125: Primitives[keyof Primitives]
+
+  blue12: Primitives[keyof Primitives]
+  blue4570: Primitives[keyof Primitives]
+  blue5782: Primitives[keyof Primitives]
+
+  green12: Primitives[keyof Primitives]
+  green4570: Primitives[keyof Primitives]
+  green5782: Primitives[keyof Primitives]
+
+  orange12: Primitives[keyof Primitives]
+  orange3045: Primitives[keyof Primitives]
+  orange4570: Primitives[keyof Primitives]
+  orange5782: Primitives[keyof Primitives]
+
+  red12: Primitives[keyof Primitives]
+  red4570: Primitives[keyof Primitives]
+  red5782: Primitives[keyof Primitives]
```

Comment:
what was the intention here? the types just resolve to `string`

Reply:
There is value in expressing more than just strings (as we agreed on with shared-types/src/ComponentThemeVariables.ts below). The correct way to type them for examle:

grey1214: Primitives['grey12'] | Primitives['grey14']